Cargo.toml

```diff
@@ -24,6 +24,10 @@ gethostname = "0.2.1"
 [target.'cfg(unix)'.dependencies]
 nix = "0.17"
 
+[target.'cfg(windows)'.dependencies.winapi]
+version = "0.3"
+features = ["winsock2"]
```

Comment:
I think that this is the best solution, have a safe wrapper in a separate crate. I think that it is best to keep it as a separate repository, so tagging versions is simpler.

Reply:
Okay. Feel free to tell me when to do a release of `winapi-wsapoll`. We could add more information to it before then, e.g. `description` and `license` in `Cargo.toml`.

What would be a good version number? The default seems to be `0.1.0`, which is okay. `1.0.0` is likely okay, but it would be weird to go 1.0 when winapi is only at `0.3`...

Reply:
I think `winapi-wsapoll` is good enough for our use case, so let's make a release now. I would use `0.1.0` because I don't feel comfortable releasing a `1.0.0` that quickly.